src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -82,6 +82,7 @@ const PackageRecommendations = () => {
           recommendPackageRequest: {
             packages: packages.map((pkg) => pkg.name),
             recommendedPackages: 5,
+            distribution: distribution,
```

Comment:
Looks great! :heart: Let's update this
```suggestion
            distribution: distribution.replace('-', ''),
```

to fix the `Recommendations couldn't be fetched` error and we're ready to merge :rocket:

You can test the recommendations for RHEL 9 with `blktrace` package, I can see some recommendations there :sparkles: